src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/apexcharts.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import ApexCharts from 'apexcharts';
+
+(function () {
```

Comment:
This is vanilla JS "modules" (revealing module pattern), which are not necessary in presence of webpack bundler as this whole file already a CommonJS module (Or some mix of CommonJs/ES modules, I'm not quite sure).